addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -426,6 +426,100 @@ QUnit.test('chatter should become enabled when creation done', async function (a
     );
 });
 
+QUnit.test('readmore/less span do not duplicate', async function (assert) {
+    assert.expect(8);
+
+    this.data['mail.message'].records = [{
+        id: 1000,
+        body: `
+            <div>
+                Dear Joel Willis,<br>
+                Thank you for your enquiry.<br>
+                If you have any questions, please let us know.
+                <br><br>
+                Thank you,<br>
+                <span data-o-mail-quote="1">-- <br data-o-mail-quote="1">
+                    System
+                </span>
+            </div>
+        `,
```

Comment:
Would put an explanation on what parts are expected to be compacted in "read more/less".
`data-o-mail-quote` seems like a processing that is being done by server, would be nice to mention that.